recipe/meta.yaml

```
@@ -40,40 +35,30 @@ outputs:
         - wheel
       run:
         - python
-        - {{ pin_subpackage('google-cloud-bigquery-core', exact=True) }}
-        # restricting to python >=3.8 to avoid dependency on importlib_metadata for python <3.8
-        # bqstorage and pandas extras
-        - pyarrow >=3.0.0
-        # bqstorage extra
-        - google-cloud-bigquery-storage >=2.6.0,<3.0.0dev
-        # following conda-forge here, the constraint is more restrictive but valid for all py versions
-        - grpcio >=1.49.1,<2.0dev
-        # pandas extra
-        - pandas >=1.1.0
-        - db-dtypes >=0.3.0,<2.0.0dev
-        # ipywidgets extra
-        - ipywidgets >=7.7.0  # [not (s390x and py>311)]
-        # - ipykernel >=6.0.0
-        # geopandas extra
-        - geopandas >=0.9.0,<1.0dev  # [not s390x]
-        - shapely >=1.8.4,<3.0.0dev  # [not s390x]
-        # ipython extra
-        - ipython >=7.23.1,!=8.1.0
-        - ipykernel >=6.0.0
-        # tqdm extra
-        - tqdm >=4.7.4,<=5.0.0dev
-        # bigquery_v2 extra
-        - proto-plus >=1.15.0,<2.0.0dev
-        - protobuf >=3.19.5,<5.0.0dev,!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5
-
+        - google-api-core-grpc >=2.11.1,<3.0.0
+        - google-auth >=2.14.1,<3.0.0
+        - google-cloud-core >=2.4.1,<3.0.0
+        - google-resumable-media >=2.0.0,<3.0
+        - packaging >=24.2.0
+        - python-dateutil >=2.8.2,<3.0
+        - requests >=2.21.0,<3.0.0
+      run_constrained:
+        - pandas >=1.1.4
+        - pandas-gbq >= 0.26.1
+        - db-dtypes >=1.0.4,<2.0.0
+        - ipywidgets >=7.7.1
+        - ipykernel >=6.2.0
+        - geopandas >=0.9.0,<2.0
+        - shapely >=1.8.4,<3.0.0
+        - ipython >=7.23.1
+        - bigquery-magics >=0.6.0
+        - tqdm >=4.7.4,<=5.0.0
+        - opentelemetry-api >=1.1.0
+        - opentelemetry-sdk >=1.1.0
+        - opentelemetry-instrumentation >=0.20b0
+        - proto-plus >=1.22.3,<2.0.0
+        - protobuf >=3.20.2,<7.0.0,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!= 4.21.5
```

```suggestion
        - protobuf >=3.20.2,<7.0.0,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!= 4.21.5
        - google-cloud-bigquery-storage >=2.18.0,<3.0.0
        - grpcio >=1.47.0,<2.0.0  # [py<311]
        - grpcio >=1.49.1,<2.0.0  # [py>=311]
        - pyarrow >=4.0.0
```
To also consider the one added by the extra output.